setup.py

```diff
@@ -28,6 +28,9 @@
         'appdirs',
         'chardet>=3.0.4',      # rarely used but small/omnipresent
         'colorama; platform_system=="Windows"',
+        # for as long as we support python 3.5
+        # to prevent "UnicodeEncodeError: 'charmap' codec can't encode..."
+        'win-unicode-console; platform_system=="Windows"',
```

Comment:
`python_requires='>=3.6'` is already declared below.

Reply:
Yeah, this lead to nothing. I removed it in full.